tests/integration/targets/aci_domain_to_encap_pool/tasks/main.yml

```diff
@@ -145,6 +151,9 @@
     vm_provider: cloudfoundry
     state: present
   register: vmm_cloudfoundry
+  when:
+  - version.current.0.topSystem.attributes.version is version('4.1', '>=')
```

Comment:
Does this fails on 3.2? This conditions are blocking testing on 3.2

Reply:
yes it fails on 3.2 hence the condition